### ISSUE TITLE
Start drag on hold, but only on touch devices.

### ISSUE THREAD

**Issue by jeffijoe** [taye/interact.js#287]
Hi, awesome library! :)

Is it possible to start a drag regularly if on desktop, but only after hold on mobile?

I saw the FAQ, but the code there will force the hold on all devices.

**Comment by taye** [comment]
You can modify the example on the FAQ so that it tries to start interactions on both `hold` and `move` events depending on the `event.pointerType`

```javascript
  .on('move hold', function (event) {
    var interaction = event.interaction;

    // don't try to start if already interacting
    if (interaction.interacting()) return;

    // if the pointer is a mouse
    if (event.pointerType === 'mouse') {
      // return early if it wasn't a move event
      if (event.type !== 'move') return;
    }
    // return early if it's not a mouse (touch/pen) and it's not a hold event
    else if (event.type !== 'hold') return;

    interaction.start({ name: 'drag' }, ...
```

**Comment by jeffijoe** [comment]
@taye thanks, but this seems to be creating elements whenever the pointer moves (using clones), how can I prevent that?

**Comment by jeffijoe** [comment]
@taye never mind that, I just copy-pasted without thinking, I forgot to check if the pointer was down and not interacting. :P

Thank you for your help!
